sdk/storage/azure-storage-blob-nio/src/main/java/com/azure/storage/blob/nio/AzureFileSystem.java

```diff
@@ -187,18 +209,30 @@ public boolean isReadOnly() {
      */
     @Override
     public String getSeparator() {
-        return "/";
+        return AzureFileSystem.PATH_SEPARATOR;
     }
 
     /**
+     * The list of root directories corresponds to the list of available file stores and therefore containers specified
+     * upon initialization. A root directory always takes the form {@code "<file-store-name>:"}. This list will not
+     * change during the lifetime of this object. If containers are added to the account after initialization, they
+     * will be ignored. If a container is deleted or otherwise becomes unavailable, its root directory will still be
+     * returned but operations to it will fail.
```

Comment:
In case we later allow users to specify an account instead of individual containers, let's change `This list will not change during the lifetime of this object. If containers are added to the account after initialization, they will be ignored.` to `Results of this list are scoped based on specifications during FileSystem initialization.`

In this same vein, we can change `If a container is deleted or otherwise becomes unavailable, its root directory will still be returned but operations to it will fail.` to `Containers listed are only guaranteed to have existed at some point in time. Operations on nonexistent containers will fail.`. This gives us some more generic language that allows us to update implementation details later on.